### PR TITLE
fix(FEC-9493): ima - dragging to the end causes the midroll to play with replay button

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

**Issue:** ima plays the midroll on `ENDED` asynchronously, so `PLAYBACK_ENDED` fired before `AD_LOADED` and `AD_BREAK_START`.
**Solution:** Hide the replay button while ad break, even `isPlaybackEnded` prop is `true`

(This is for ima vmap. The same issue with ad layout fixed by https://github.com/kaltura/playkit-js/pull/416)

Solves [FEC-9493](https://kaltura.atlassian.net/browse/FEC-9493)

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/components/play-pause/play-pause.js [hunk 1]
```diff
@@ -62,11 +62,19 @@ class PlayPause extends Component {
           <button
             tabIndex="0"
             aria-label={
-              <Text id={props.isPlaybackEnded ? 'controls.startOver' : this.props.isPlayingAdOrPlayback ? 'controls.pause' : 'controls.play'} />
+              <Text
+                id={
+                  props.isPlaybackEnded && !this.props.adBreak
```
Comment (RoyBregman) [inline src/components/play-pause/play-pause.js]: extract to const isStartOver to be used also in line 77

**Comment by OrenMe** [comment]
How does this happen @yairans ? playback ended should happen only after all ads were played, so the new condition is inherently incorrect.
So it looks like we have some other issue which is not necessarily in the UI - where is it?

**Comment by yairans** [comment]
@OrenMe as I wrote in the description, ima triggers the midroll on ended **asynchronously**. We don’t know in our ended handler that an ad will play soon and we trigger playback-ended

File: src/components/play-pause/play-pause.js (hunk 1 above)
Comment (yairans) [reply src/components/play-pause/play-pause.js]: done

**Pull request merged**